src/main/webapp/app/overview/course-exams/course-exam-detail/course-exam-detail.component.html

```diff
@@ -1,15 +1,88 @@
+            </h4>
+            <div class="col-8 row card-body justify-content-center">
+                <div *ngIf="examState === 'UPCOMING'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'upcoming' | artemisTranslate }}</h5>
+                    <div class="text-center">
+                        {{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'imminent' | artemisTranslate }}
+                        {{ timeLeftToStart | artemisDurationFromSeconds }}
+                    </div>
+                </div>
+                <div *ngIf="examState === 'IMMINENT'">
+                    <h5 class="text-center">
+                        {{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'imminent' | artemisTranslate }}
+                        {{ timeLeftToStart | artemisDurationFromSeconds }}
+                    </h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'imminentExplanation' | artemisTranslate }}</div>
+                </div>
+                <div *ngIf="examState === 'CONDUCTING'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'conducting' | artemisTranslate }}</h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'conductingExplanation' | artemisTranslate }}</div>
+                </div>
+                <div *ngIf="examState === 'TIMEEXTENSION'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.timeExtension' | artemisTranslate }}</h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.timeExtensionExplanation' | artemisTranslate }}</div>
+                </div>
+                <div *ngIf="examState === 'CLOSED'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'closed' | artemisTranslate }}</h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.' + (exam.testExam ? 'testExam.' : '') + 'closedExplanation' | artemisTranslate }}</div>
+                </div>
+                <div *ngIf="examState === 'STUDENTREVIEW'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.review' | artemisTranslate }}</h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.reviewExplanation' | artemisTranslate }} {{ exam.examStudentReviewEnd | artemisDate }}</div>
+                </div>
+                <div *ngIf="examState === 'NO_MORE_ATTEMPTS'">
+                    <h5 class="text-center">{{ 'artemisApp.exam.overview.noMoreAttempts' | artemisTranslate }}</h5>
+                    <div class="text-center">{{ 'artemisApp.exam.overview.noMoreAttemptsExplanation' | artemisTranslate }}</div>
+                </div>
+            </div>
+        </div>
```

Comment:
Please consider making use of [NgSwitch](https://angular.io/api/common/NgSwitch) for cases like these :)

Reply:
Thanks for the hint 👍